### PR TITLE
Json Back - Upgrade

### PR DESCRIPTION
**Pull request by SpliiT**

### CODE REVIEW

File: geode_objects.py
Comment (JulienChampagnol) [inline geode_objects.py:1]: To delete

File: blueprints/blueprint_workflows.py [hunk 1]
```diff
@@ -7,21 +7,33 @@
 import blueprints.workflows.blueprint_simplex as bp_simplex
 import blueprints.workflows.blueprint_explicit as bp_explicit
 
-workflows_routes = flask.Blueprint('crs_converter_routes', __name__)
+workflows_routes = flask.Blueprint("crs_converter_routes", __name__)
```
Comment (JulienChampagnol) [inline blueprints/blueprint_workflows.py:10]:
```suggestion
workflows_routes = flask.Blueprint("workflows_routes", __name__)
```

**Review by JulienChampagnol — APPROVED** [review]

File: blueprints/tools/crs_converter_allowed_files.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "$id": "/tools/crs_converter/allowed_files",
+    "route": "/allowed_files",
+    "methods": [
+        "GET"
+    ],
+    "type": "object",
+    "properties": {},
```
Comment (BotellaA) [inline blueprints/tools/crs_converter_allowed_files.json:8]: to remove.... etc

File: blueprints/workflows/implicit_step0.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "$id": "/workflows/implicit/step0",
+  "route": "/step0",
+  "methods": [
+    "POST"
+  ],
+  "type": "object",
+  "properties": {},
+  "required": [],
```
Comment (JulienChampagnol) [inline blueprints/workflows/implicit_step0.json:9]: To remove ?
```suggestion
```

**Review by JulienChampagnol — APPROVED** [review]

**Comment by JulienChampagnol** [comment]
Think about updating your dependencies after publishing the OGW-front package

**Pull request merged**